Tasks/AppCenterDistribute/task.json

```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 0,
         "Minor": 131,
```

Comment:
Even if this isn't delivering a new feature, I would probably bump Minor to 134 (the current VSTS sprint), or 133 if you're going to port to the release branch, instead of bumping patch. That gives a rough indication of when the task was last modified.

Reply:
Thanks for the review, bumped the version.